disnake/message.py

```diff
@@ -1250,6 +1291,17 @@ def thread(self) -> Optional[Thread]:
 
         return self.guild.get_thread(self.id)
 
+    @property
```

Comment:
use cached_property? or is RoleSubscriptionData mutable by the user

Reply:
I shouldn't be mutated, but is it worth storing this in memory for the lifetime of the Message object?

Reply:
... fair, messages can be cached for a long time.